main.go

```diff
@@ -62,9 +65,9 @@ func main() {
 		SendBufferQueue: sendBufferQueue,
 	}
 	writer := &Writer{
-		SendBufferQueue:   sendBufferQueue,
```

Comment:
Yeah, I'm not sure what introduced the whitespace changes here. I don't recall doing it. I considered reverting it, but decided not to bother instead.